tests/installation/first_boot.pm

```diff
@@ -10,6 +10,11 @@ sub run() {
         return;
     }
 
+    if (check_var('DESKTOP', 'awesome')) {
```

Comment:
@nilxam thanks for your comments. Maybe I can update the commits with the comments and merge so that we don't have to wait and @asdil12 does not have to do another iteration.